GUIDELINES.md

```diff
@@ -72,13 +72,12 @@ recipes. Recipes for dependencies with an `r-` prefix should be created using
 
 Add a wrapper script if the software is typically called via `java -jar ...`.
 For example, [fastqc](recipes/fastqc) already has a wrapper script, but
-[gatk-framework](recipes/gatk-framework) does not. [chromhmm](recipes/chromhmm)
-mimics the installation pattern of gatk-framework.
+[peptide-shaker](recipes/peptide-shaker) does not.
```

Comment:
Yes, what this means is that peptide-shaker has a wrapper that we had to provide. Fastqc provides its own wrapper.